contracts/mod-balances/Cargo.toml

```
@@ -19,7 +19,10 @@ cosmwasm-std = { workspace = true }
 cosmwasm-schema = { workspace = true }
 cw2 = { workspace = true }
 cw20 = { workspace = true }
-thiserror = { workspace = true }
+cw20-base = { workspace = true }
+cw-multi-test = { workspace = true }
+cw-utils = { workspace = true }
 mod-sdk = { workspace = true }
+thiserror = { workspace = true }
 
 [dev-dependencies]
```

```suggestion
mod-sdk = { workspace = true }
thiserror = { workspace = true }

[dev-dependencies]
cw20-base = { workspace = true }
cw-multi-test = { workspace = true }
cw-utils = { workspace = true }
```
We don't test dependencies to be included outside tests